cg_hermes/config/tags.py

```diff
@@ -112,6 +112,7 @@
     "tcov": {"description": "Coverage output"},
     "tiddit-coverage": {"description": "Coverage output from tiddit"},
     "visualization": {"description": "Visualizes data"},
+    "umi": {"description": "UMI workflow"}
```

Comment:
How about this if it is more clear to understand?

```suggestion
    "umi": {"description": "Files related to UMI workflow"}
```